Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -102,6 +102,9 @@ class HTTPClient {
 
     // Visible for tests
     var defaultHeaders: RequestHeaders {
+        let supportedLocales = self.systemInfo.preferredLanguages.map {
+            $0.replacingOccurrences(of: "-", with: "_")
+        }.joined(separator: ",")
```

Comment:
I would add a maximum here... Maybe first 3? Just so this header can't grow indefinitely.